Package.swift

```diff
@@ -16,11 +16,15 @@ let package = Package(
         ),
     ],
     dependencies: [
+        .package(url: "https://github.com/stephencelis/SQLite.swift.git", exact: Version(0, 13, 3)),
```

Comment:
💡 Future consideration; you might wanna eventually move the SQL based storage code to a separate repo so people who aren't using it dont have to pull SQLite.swift etc.. in as unneeded deps

Reply:
I'd considered that but now that `SQLiteStorageEngine` is the default choice I don't see a straightforward way to do that?

Reply:
yeah just wanted to call it out as an option. you'd need to make another repo/package that depends on Bodega and is just the SQL engine